extensions/notebook/src/test/book/book.test.ts

```diff
@@ -265,29 +268,29 @@ describe('BookTreeViewProviderTests', function() {
 
 		it('should show error message if config.yml file not found', async () => {
 			await bookTreeViewProvider.currentBook.readBooks();
-			should(bookTreeViewProvider.errorMessage.toLocaleLowerCase()).equal(('ENOENT: no such file or directory, open \'' + configFile + '\'').toLocaleLowerCase());
+			should(bookTreeViewProvider.currentBook.errorMessage.toLocaleLowerCase()).equal(('ENOENT: no such file or directory, open \'' + configFile + '\'').toLocaleLowerCase());
 		});
 
-		it('should show error if toc.yml file format is invalid', async function(): Promise<void> {
+		it('should show error if toc.yml file format is invalid', async function (): Promise<void> {
 			await fs.writeFile(configFile, 'title: Test Book');
 			await bookTreeViewProvider.currentBook.readBooks();
-			should(bookTreeViewProvider.errorMessage).equal('Error: Test Book has an incorrect toc.yml file');
+			should(bookTreeViewProvider.currentBook.errorMessage).equal('Invalid toc file');
```

Comment:
Why did we change the error message? The old one seems more useful

Reply:
Test Book is the book title and that information is no longer available when it fails in the current flow.